sidekicks/schemas.yaml

```
@@ -174,4 +174,22 @@ Learn:
       description: |
         Number of times a new Sidekick has been learned on this boot.
         Useful in UI design to know that the learn process completed,
-        even if the learned address is the same.
\ No newline at end of file
+        even if the learned address is the same.
+    new_id:
+      example: "SMXXX12345"
+      type: string
+      readOnly: true
+      description: |
+        [v4.24+] ID of the newly learned Sidekick, if any.
+        This field is cleared when opening a new learn window.
+        Populated when a new Sidekick successfully links during the learn window.
```

This has existed for some time, so I would move the tag for the clearing, which is new.
```suggestion
        ID of the newly learned Sidekick, if any.
        Populated when a new Sidekick successfully links during the learn window.
        [v4.24+] This field is cleared when opening a new learn window.
```